### ISSUE TITLE
app.page.pop() not working in iOS

### ISSUE THREAD

**Issue by RoboKopf** [Mobideck/appdeck#62]
I'm having a problem with app.page.pop() on iOS. It does nothing on either your emulator or your iOS app. It ought to do exactly what tapping the close icon does, no?

I have been able to use app.page.root('page.php') to close the pushed page and return to the root page, but this is undesirable because it reloads the root page instead of simply closing the pushed page. This also interferes with passing a message to the root page from the pushed page just before closing the pushed page.

In short, I would like app.page.pop() to work the same on iOS as it does on android: close the pushed page, and nothing more or less.

Thank you!

**Comment by RoboKopf** [comment]
Also: I cannot find a to-do list for appdeck. Would you be willing to make your roadmap/to-do list public?

**Comment by mdekermadec** [comment]
Does this works in the kitchensink ?

I just made a test on my iPhone 5S and it was OK. This may come from either a specific bug on your kinf of iOS device or there may be a uncovered case in the api.

Could you give me your Javascript code ?

About Roadmap, this is a good idea. I try to setup something in the wiki.
